files/preStudies/frameworks.tex

```diff
@@ -0,0 +1,80 @@
+\section{Frameworks}
+
+This section will evaluate different frameworks for cross-platform development which could be used in this project. Some must have and nice to have capabilities will be
```

Comment:
"Must have" and Nice to have" are terms and should be in italics. 
The sentence is not finished?